docs/release/trg-0/trg-9-02.md

```diff
@@ -0,0 +1,154 @@
+---
+title: TRG 9.02 - KIT Maturity Levels
+---
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 119-Feb-2024  | Initial contribution                   |
+
+## KIT Maturity Levels
+
+KITs have a maturity level of sandbox, incubating, or graduated.
+
+The maturity levels of KITs signals the quality, completeness and proof that the use case solves specific business problem and provides value to all stakeholders.
+
+In order to be promoted from one level to the next, the developers must fulfill various criteria. For each stage, different artifacts and deliverables are defined that must be completed. A distinction is made between mandatory (✓) and recommended ((✓)) deliverables. In order to be promoted to the next level, a kit must have completed all necessary items of the lower level and at least one necessary item of the following level. As soon as this criteria is met a promotion has to be requested. To achieve graduated status, additionally a case study must be completed with a partner from the community. The aim of this case study is to verify the content of the KIT. The performing partner tries to solve the problem only with the help of the KIT and, if necessary, to make suggestions for improvement that have to be implemented.
+
+## Initiating a new KIT
```

Comment:
I don't think this is the right place for this. A TRG is a list of clear rules you need to follow when you do a quality check.

This is a process.

The process and the overall KIT handling is discussed already here https://eclipse-tractusx.github.io/docs/kit-process/graduation-process

Reply:
We will delete the whole KIT guidelines section with a new, separated PR.